api/v1alpha1/common_types.go

```diff
@@ -32,8 +32,13 @@ type ClusterReference struct {
 	Name      string `json:"name"`
 	Namespace string `json:"namespace,omitempty"`
 }
+type CruiseControlTaskOperation string
 
 const (
+	// OperationAddBroker means a Cruise Control add_broker operation
+	OperationAddBroker CruiseControlTaskOperation = "addbroker"
+	// OperationRemoveBroker means a Cruise Control remove_broker operation
+	OperationRemoveBroker CruiseControlTaskOperation = "removebroker"
```

Comment:
Shouldn't the const variables be `"addBroker"` and `"removeBroker"`?

Reply:
I think it would be better to change this to add_broker, remove_broker, ... like here: https://github.com/linkedin/cruise-control/wiki/REST-APIs#post-requests. Later it can be changed. Im not sure yet.

Reply:
I have changed to add_broker, remove_broker , and  I have added a new one "rebalance"

Reply:
I agree with using the CC terminology here to avoid unnecessary conversion requirements between enums/strings on component boundaries.

Reply:
thanks for the info @bartam1